com.unity.render-pipelines.universal/ShaderLibrary/ShaderVariablesFunctions.hlsl

```diff
@@ -283,4 +284,33 @@ float2 GetNormalizedScreenSpaceUV(float4 positionCS)
     #define UnityStereoTransformScreenSpaceTex(uv) uv
 #endif // defined(UNITY_SINGLE_PASS_STEREO)
 
+uint2 ComputeFadeMaskSeed(float3 V, uint2 positionSS)
```

Comment:
this function is already in LODCrossFade.hlsl, is there a reason to have it twice ?

Reply:
It's defined in HDRP's ShaderVariableFunctions but not in Universal, so when I define it in LODCrossfade I get redefinition errors. 
So I moved the definition to Universal's ShaderVariableFunctions instead.